state/helpers_test.go

```diff
@@ -54,7 +55,12 @@ func makeAndCommitGoodBlock(
 
 func makeAndApplyGoodBlock(state sm.State, height int64, lastCommit *types.Commit, proposerAddr []byte,
 	blockExec *sm.BlockExecutor, evidence []types.Evidence) (sm.State, types.BlockID, error) {
-	block, _ := state.MakeBlock(height, makeTxs(height), lastCommit, evidence, proposerAddr)
+	block, _ := state.MakeBlock(
+		height,
+		factory.MakeData(factory.MakeTenTxs(height), evidence, nil),
```

Comment:
```suggestion
		factory.MakeData(makeTxs(height), evidence, nil),
```
[minor] ... unless there was a reason other than mechanically adapting the tests